src/image.ts

```diff
@@ -0,0 +1,50 @@
+import {parse} from 'csv-parse/sync';
+
+export interface Image {
+  name: string;
+  enable: boolean;
+}
+
+export function Transform(inputs: string[]): Image[] {
+  const images: Image[] = [];
+  for (const input of inputs) {
+    const image: Image = {name: '', enable: true};
+    const fields = parse(input, {
+      relaxColumnCount: true,
+      skipEmptyLines: true
+    })[0];
+    for (const field of fields) {
+      const parts = field
+        .toString()
+        .split('=')
```

Comment:
I think splitting by `=` makes something like this invalid 
```
name=lucacome/test-foo2,enable=${{ github.event_name == 'pull_request' }}
```
not sure if you ever meant for it to be supported and I guess I could always have the same condition with other operators like `startsWith` without using `=`

Reply:
`${{ github.event_name == 'pull_request' }}` is interpreted upstream by the GitHub Runner logic so it would produce `name=lucacome/test-foo2,enable=<true|false>` before being passed to the action.

Reply:
ah interesting, I thought that was the cause for `Error: Image name attribute empty: enable=true` that I'm seeing, but then I'm not sure I understand what this error means 😄 

I have this and I assumed it wasn't coming from the first one, as it would get the default `enable=true`, and that it was the second line causing the error.
```
images: |
  name=lucacome/test-foo
  name=lucacome/test-foo2,enable=${{ github.event_name == 'pull_request' }}
  name=ghcr.io/lucacome/workflows,enable=${{ startsWith(github.ref, 'refs/tags/') && contains(matrix.target, 'aws') }}
 ```

Reply:
Oh sounds like a bug then, will take a look, thanks for your feedback

Reply:
@lucacome Hum I was not able to repro, do you have a link to your repo?

Reply:
@crazy-max I have this repo where I tests workflows https://github.com/lucacome/workflows/runs/6178399824?check_suite_focus=true

Reply:
Ok was able to repro: https://github.com/docker/metadata-action/runs/6195927044?check_suite_focus=true#step:3:19

Reply:
@lucacome Should be fixed, let me know if it looks good to you. Thanks!

Reply:
LGTM! 🚀